src/js/components/TextInput/stories/textinput.stories.js

```diff
@@ -84,7 +85,79 @@ class SuggestionsTextInput extends Component {
   }
 }
 
+const customTheme = deepMerge(grommet, {
+  textInput: {
+    extend: () => `
+      font-size: 20px;
+      background: #c9c19f;
+      width: 300px;
+      margin: 0 auto;
+      
+      &:focus {
+        box-shadow: none;
+        border-color: initial;
+      }
+    `,
+    container: {
+      extend: () => `
+        background: #edf7d2;
+        height: 100px;
+        width: 400px;
+        display: flex;
+        flex-flow: column;
+        justify-content: center;
+        border-radius: 10px;
+      `,
+    },
+    placeholder: {
+      extend: () => `
+        width: 100%;
+        color: #1e1a11;
+      `,
+    },
+    suggestions: {
+      extend: () => `
+        background: #c9c19f;
+        color: #3d3522;
+        li {
+          border-bottom: 1px solid rgba(0, 0, 0, 0.2);
+        }
+      `,
+    },
+  },
+});
+
+class ThemedTextInput extends Component {
+  state = { value: '' };
+
+  onChange = event => this.setState({ value: event.target.value });
+
+  onSelect = event => this.setState({ value: event.suggestion });
+
+  render() {
+    const { value } = this.state;
+    return (
+      <Grommet full theme={customTheme}>
+        <Box fill align="center" justify="start" pad="large">
+          <Box width="medium">
+            <TextInput
+              type="password"
+              value={value}
+              dropProps={{ height: 'small' }}
+              onChange={this.onChange}
+              onSelect={this.onSelect}
+              suggestions={suggestions}
+              placeholder={<span>Enter something...</span>}
+            />
+          </Box>
+        </Box>
+      </Grommet>
+    );
+  }
+}
+
 storiesOf('TextInput', module)
   .add('Simple TextInput', () => <SimpleTextInput />)
   .add('Password input', () => <PasswordInput />)
-  .add('Suggestions TextInput', () => <SuggestionsTextInput />);
+  .add('Suggestions TextInput', () => <SuggestionsTextInput />)
```

Comment:
We are trying to avoid re-stating the name of the component, so this should work:
```suggestion
  .add('Suggestions', () => <SuggestionsTextInput />)
```